web/client/plugins/FeatureEditor.jsx

```diff
@@ -111,7 +111,17 @@ const Dock = connect(createSelector(
   *         }
   *       }]
   *     },
-  *   "editingAllowedRoles": ["ADMIN"]
+  *   "editingAllowedRoles": ["ADMIN"],
+  *   "snapTool": true,
+  *   "snapConfig": {
+  *     "vertex": true,
+  *     "edge": true,
+  *     "pixelTolerance": 10,
+ *      "additionalLayers": [
+ *         "ADDITIONAL_LAYER_ID"
+ *      ],
```

Comment:
is this still necessary?

Reply:
Yes, this one can be useful if someone needs to snap to the layer that is not available in TOC. Like urbanisme layer with recent changes. This is configurable only in localConfig and will bring layer into snapping options list if additional layer is active & supported type (vector / wms + wfs)